superset-frontend/src/explore/components/controls/ControlPopover/ControlPopover.test.tsx

```diff
@@ -89,8 +80,8 @@ test('Should lock the vertical scroll when the popover is visible', () => {
   );
 });
 
-test('Should place popover at the top', async () => {
-  const { setStateMock } = setupTest({
+test.skip('Should place popover at the top', async () => {
```

Comment:
Maybe these skipped tests are a concern... or not... perhapse I'll pull in this file from `master` and see if CI still passes, if we're concerned here.